department-of-veterans-affairs/caseflow
# Display decision date for AMA appeals in search

Resolves #7575.

When we [create BvaDispatchTasks](https://github.com/department-of-veterans-affairs/caseflow/blob/505a496eef2b77eeb0bd6726d68b2e86050a37c6/app/models/tasks/bva_dispatch_task.rb#L5), we do not set the action of those tasks to "review". Removing that constraint returns a task for decided appeals.

Currently there are 14 decided AMA appeals but when the query is constrained on the action column we return none of them. Ran a quick confirmation in the prod rails console:
```
rails c> BvaDispatchTask.where(action: "review", status: "completed").count
=> 0
rails c> BvaDispatchTask.where(status: "completed").count
=> 14
```

## Test plan:
* Locally create a decided appeal:
```
rails c> a = Appeal.find(5)
rails c> t = BvaDispatchTask.create!(appeal: a, status: Constants.TASK_STATUSES.completed, assigned_to: BvaDispatch.singleton)
rails c> t.update!(completed_at: t.created_at)
```
* Search for Veteran ID 228081153
* Notice that the decision date now appears in the table for the AMA appeal.
